docs/HOWTOS.md

```diff
@@ -0,0 +1,84 @@
+# How-to and Frequently Asked Questions
+
+This page collects information to configure and use specific features of zAppBuild. Available How-tos:
+
+- [How-to and Frequently Asked Questions](#how-to-and-frequently-asked-questions)
+  - [Signing load modules and program objects](#signing-load-modules-and-program-objects)
+
+
+## Signing load modules and program objects
+
+zAppBuild is configured to automatically insert a reference of the version of the source code (the githash), which was used to build the executables. At the moment, the feature is only available for build files which are mapped to `Assembler.groovy`, `Cobol.groovy` and `PLI.groovy`. It is available for pipeline builds.
+
+It leverages the [IDENTIFY statement](https://www.ibm.com/docs/en/zos/2.5.0?topic=reference-identify-statement) of the linkage editor.
+
+**How to enable the signing**
+
+The feature is enabled by default and is controlled via the `assembler_identifyLoad`, `cobol_identifyLoad` and the `cobol_identifyLoad` properties managed the corresponding property files of the language scripts.
```

Comment:
I believe a word is missing at the end of the sentence: "managed by" or "managed in" ?